SonarQube.Client.Tests/Requests/Api/V7_20/GetIssuesRequestTests.cs

```diff
@@ -0,0 +1,233 @@
+            }
+          ],
+        }
+     ],
+      ""status"": ""OPEN"",
+      ""message"": ""Refactor this code to not perform redirects based on tainted, user-controlled data."",
+      ""effort"": ""30min"",
+      ""debt"": ""30min"",
+      ""assignee"": ""rita-g-sonarsource@github"",
+      ""author"": ""rita.gorokhod@sonarsource.com"",
+      ""tags"": [],
+      ""creationDate"": ""2020-07-16T21:31:25+0200"",
+      ""updateDate"": ""2020-07-16T21:34:05+0200"",
+      ""type"": ""VULNERABILITY"",
+      ""organization"": ""myorganization"",
+      ""fromHotspot"": false
+    }
+  ],
+  ""components"": [
+    {
+      ""organization"": ""myorganization"",
+      ""key"": ""myprojectkey:projectroot/Controllers/WeatherForecastController.cs"",
+      ""uuid"": ""AXNZHtnVuQ67pPQjI7ey"",
+      ""enabled"": true,
+      ""qualifier"": ""FIL"",
+      ""name"": ""WeatherForecastController.cs"",
+      ""longName"": ""projectroot/Controllers/WeatherForecastController.cs"",
+      ""path"": ""projectroot/Controllers/WeatherForecastController.cs""
+    },
+    {
+      ""organization"": ""myorganization"",
+      ""key"": ""myprojectkey:projectroot/Controllers/Helper.cs"",
+      ""uuid"": ""AXNZHtnVuQ67pPQjI7ez"",
+      ""enabled"": true,
+      ""qualifier"": ""FIL"",
+      ""name"": ""Helper.cs"",
+      ""longName"": ""projectroot/Controllers/Helper.cs"",
+      ""path"": ""projectroot/Controllers/Helper.cs""
+    },
+    {
+      ""organization"": ""myorganization"",
+      ""key"": ""myprojectkey"",
+      ""uuid"": ""AXJLrCxxxeWiK2BCzDif"",
+      ""enabled"": true,
+      ""qualifier"": ""TRK"",
+      ""name"": ""sanity-connected"",
+      ""longName"": ""sanity-connected""
+    }
+  ],
+  ""organizations"": [
+    {
+      ""key"": ""myorganization"",
+      ""name"": ""a user""
+    }
+  ],
+  ""facets"": []
+}
+";
+
+            SetupHttpRequest(handlerMock, request, response);
+
+            var result = await testSubject.InvokeAsync(httpClient, CancellationToken.None);
+            result.Should().HaveCount(1);
+
+            result[0].RuleId.Should().Be("roslyn.sonaranalyzer.security.cs:S5146");
+            result[0].Flows.Count().Should().Be(2);
+
+            result[0].Flows[0].Locations.Count().Should().Be(3);
+            result[0].Flows[1].Locations.Count().Should().Be(2);
+
+            var location_0_0_0 = result[0].Flows[0].Locations[0];
+            location_0_0_0.Component.Should().Be("myprojectkey:projectroot/Controllers/WeatherForecastController.cs");
+            location_0_0_0.Message.Should().Be("sink: tainted value is used to perform a security-sensitive operation");
+            location_0_0_0.TextRange.StartLine.Should().Be(43);
+            location_0_0_0.TextRange.EndLine.Should().Be(43);
+            location_0_0_0.TextRange.StartOffset.Should().Be(19);
+            location_0_0_0.TextRange.EndOffset.Should().Be(43);
+
+            var location_0_1_1 = result[0].Flows[1].Locations[1];
```

Comment:
I think the name is complicated to read. 
Since there is only 1 issue, you can extract `result[0]` into a variable, and then name these `firstFlowLocation` and `secondFlowLocation` or something similar. I think it would help with readability.